connectors_service.gemspec

```diff
@@ -0,0 +1,18 @@
+require_relative 'lib/app/config'
```

Comment:
It's an implicit dependence. I wouldn't recommend making gemspec file dependent on app config or any other dependencies. 

The very first step of Bundler work is resolving dependencies by reader gemspec file. So if you run `bundle install` in the environment where `App::Config` is not available it will fail.

Reply:
This was actually copied from [connectors_utility.gemspec](https://github.com/elastic/connectors-ruby/blob/main/connectors_utility.gemspec), and I also see similar `require` in other gems like [rake](https://github.com/ruby/rake/blob/30fbf4e05bd783d4d56d9829830c2847a122e3ad/rake.gemspec#L2)

Anyway, I'm removing it as it's not needed for the connector service gem.